packages/jest-cli/src/reporters/JestReporterDefault.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+import type {AggregatedResult, TestResult} from 'types/TestResult';
+import type {Config} from 'types/Config';
+
+const JestReporterBase = require('./JestReporterBase');
+const chalk = require('chalk');
+const formatFailureMessage = require('jest-util').formatFailureMessage;
+// const Notifier = require('../Notifier');
+const path = require('path');
+
+// Explicitly reset for these messages since they can get written out in the
+// middle of error logging (should have listened to Spengler and not crossed the
```

Comment:
Can we remove whatever this thing in the parenthesis is? Who snuck this in? Oh gosh I hate comments.

@zpao is this your doing?